entity-framework/core/providers/sqlite/functions.md

```diff
@@ -74,30 +75,46 @@ Math.Round(d, digits) | round(@d, @digits)
-stringValue.ToLower()                     | lower(@stringValue)
-stringValue.ToUpper()                     | upper(@stringValue)
-stringValue.Trim()                        | trim(@stringValue)
-stringValue.Trim(trimChar)                | trim(@stringValue, @trimChar)
-stringValue.TrimEnd()                     | rtrim(@stringValue)
-stringValue.TrimEnd(trimChar)             | rtrim(@stringValue, @trimChar)
-stringValue.TrimStart()                   | ltrim(@stringValue)
-stringValue.TrimStart(trimChar)           | ltrim(@stringValue, @trimChar)
+.NET                                                         | SQL                                                    | Added in
+------------------------------------------------------------ | ------------------------------------------------------ | --------
+EF.Functions.Collate(operand, collation)                     | @operand COLLATE @collation                            | EF Core 5.0
+EF.Functions.Like(matchExpression, pattern)                  | @matchExpression LIKE @pattern
+EF.Functions.Like(matchExpression, pattern, escapeCharacter) | @matchExpression LIKE @pattern ESCAPE @escapeCharacter
+string.Compare(strA, strB)                                   | CASE WHEN @strA = @strB THEN 0 ... END
+string.Concat(str0, str1)                                    | @str0 \|\| @str1
+string.IsNullOrEmpty(value)                                  | @value IS NULL OR @value = ''
+string.IsNullOrWhiteSpace(value)                             | @value IS NULL OR trim(@value) = ''
+stringValue.CompareTo(strB)                                  | CASE WHEN @stringValue = @strB THEN 0 ... END
+stringValue.Contains(value)                                  | instr(@stringValue, @value) > 0
+stringValue.EndsWith(value)                                  | @stringValue LIKE '%' \|\| @value
+stringValue.FirstOrDefault()                                 | substr(@stringValue, 1, 1)                             | EF Core 5.0
+stringValue.IndexOf(value)                                   | instr(@stringValue, @value) - 1
+stringValue.LastOrDefault()                                  | substr(@stringValue, length(@stringValue), 1)          | EF Core 5.0
+stringValue.Length                                           | length(@stringValue)
+stringValue.Replace(oldValue, newValue)                      | replace(@stringValue, @oldValue, @newValue)
+stringValue.StartsWith(value)                                | @stringValue LIKE @value \|\| '%'
+stringValue.Substring(startIndex, length)                    | substr(@stringValue, @startIndex + 1, @length)
+stringValue.ToLower()                                        | lower(@stringValue)
+stringValue.ToUpper()                                        | upper(@stringValue)
+stringValue.Trim()                                           | trim(@stringValue)
+stringValue.Trim(trimChar)                                   | trim(@stringValue, @trimChar)
```

Comment:
We don't actually translate this for SQL Server, do we? See #22927.

Same for TrimStart/TrimEnd below

Reply:
Oh, I see now that this is the Sqlite page - please ignore (was thrown off by the PR title)

Reply:
Realized I was missing the *Relational* mappings, so SQLite got updated in this one too.